### DISCUSSION TITLE
[Backup] S3 can I custom folder for backup?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2890] [Q&A] [answered]
It's possible ? 😬

**Comment by ganigeorgiev** [comment, marked as answer]
No, only the bucket can be specified at the moment. Technically S3 don't have folders/directories and usually the "directory" is just a prefix string part of the full object key.

It is recommended to use a unique bucket per application as it is less likely to accidentally mess something in case there is a bug in PocketBase or in the other applications that uses the same storage.
